std/variant.d

```diff
@@ -3005,3 +3019,33 @@ if (isAlgebraic!VariantType && Handler.length > 0)
     auto b = v | s;
     assert(b == 3);
 }
+
+@system unittest
```

Comment:
Can these be `@safe`?

Reply:
I am afraid not. The function pointer taking care of the type magic is `@system`, as it might call some `core.stdc.*` functions like memcpy. It can be made `@trusted`, but it might be out of scope for this issue.